docs/api/app.md

```diff
@@ -603,6 +603,11 @@ To set the locale, you'll want to use a command line switch at app startup, whic
 
 **Note:** On Windows you have to call it after the `ready` events gets emitted.
 
+### `app.getLocaleCountryCode()`
+Returns `string` - User operating system´s locale country code in ISO3166 [here](https://www.iso.org/iso-3166-country-codes.html). The value is taken from native OS APIs.
```

Comment:
Please make "ISO3166" a link and remove the "here" word. The text looks weird now.
```suggestion
Returns `string` - User operating system's locale two-letter [ISO 3166](https://www.iso.org/iso-3166-country-codes.html) country code. The value is taken from native OS APIs.
```